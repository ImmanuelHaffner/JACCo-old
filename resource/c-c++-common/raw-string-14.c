




const void *s0 = R"abcdef()abcdef";



const void *s1 = R"??/
()??/";




const void *s2 = R"abcdefghijklmn??/(a)abcdefghijklmn???";



const void *s3 = R"abcdefghijklmno??/(a)abcdefghijklmno???";


const void *s4 = R"abcdefghijklmnop??=(a)abcdefghijklmnop??=";


const void *s5 = R"abc()abcdef";



const void *s6 = R"()";



const void *s7 = R"a()a";




int main () {}
