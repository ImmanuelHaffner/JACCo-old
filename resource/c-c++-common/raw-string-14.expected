0
raw-string-14.c:6:1 keyword const
raw-string-14.c:6:7 keyword void
raw-string-14.c:6:12 punctuator *
raw-string-14.c:6:13 identifier s0
raw-string-14.c:6:16 punctuator =
raw-string-14.c:6:18 identifier R
raw-string-14.c:6:19 string-literal "abcdef()abcdef"
raw-string-14.c:6:35 punctuator ;
raw-string-14.c:10:1 keyword const
raw-string-14.c:10:7 keyword void
raw-string-14.c:10:12 punctuator *
raw-string-14.c:10:13 identifier s1
raw-string-14.c:10:16 punctuator =
raw-string-14.c:10:18 identifier R
raw-string-14.c:10:19: warning: missing terminating " character [enabled by default]
 const void *s1 = R"??/
                   ^
raw-string-14.c:10:19 unknown "??/
raw-string-14.c:10:20: warning: trigraph ??/ ignored, use -trigraphs to enable [-Wtrigraphs]
 const void *s1 = R"??/
 ^
raw-string-14.c:11:1 punctuator (
raw-string-14.c:11:2 punctuator )
raw-string-14.c:11:3: warning: trigraph ??/ ignored, use -trigraphs to enable [-Wtrigraphs]
 ()??/";
 ^
raw-string-14.c:11:3 punctuator ?
raw-string-14.c:11:4 punctuator ?
raw-string-14.c:11:5 punctuator /
raw-string-14.c:11:6: warning: missing terminating " character [enabled by default]
 ()??/";
      ^
raw-string-14.c:11:6 unknown ";
raw-string-14.c:16:1 keyword const
raw-string-14.c:16:7 keyword void
raw-string-14.c:16:12 punctuator *
raw-string-14.c:16:13 identifier s2
raw-string-14.c:16:16 punctuator =
raw-string-14.c:16:18 identifier R
raw-string-14.c:16:19 string-literal "abcdefghijklmn??/(a)abcdefghijklmn???"
raw-string-14.c:16:34: warning: trigraph ??/ ignored, use -trigraphs to enable [-Wtrigraphs]
 const void *s2 = R"abcdefghijklmn??/(a)abcdefghijklmn???";
 ^
raw-string-14.c:16:58 punctuator ;
raw-string-14.c:20:1 keyword const
raw-string-14.c:20:7 keyword void
raw-string-14.c:20:12 punctuator *
raw-string-14.c:20:13 identifier s3
raw-string-14.c:20:16 punctuator =
raw-string-14.c:20:18 identifier R
raw-string-14.c:20:19 string-literal "abcdefghijklmno??/(a)abcdefghijklmno???"
raw-string-14.c:20:35: warning: trigraph ??/ ignored, use -trigraphs to enable [-Wtrigraphs]
 const void *s3 = R"abcdefghijklmno??/(a)abcdefghijklmno???";
 ^
raw-string-14.c:20:60 punctuator ;
raw-string-14.c:23:1 keyword const
raw-string-14.c:23:7 keyword void
raw-string-14.c:23:12 punctuator *
raw-string-14.c:23:13 identifier s4
raw-string-14.c:23:16 punctuator =
raw-string-14.c:23:18 identifier R
raw-string-14.c:23:19 string-literal "abcdefghijklmnop??=(a)abcdefghijklmnop??="
raw-string-14.c:23:36: warning: trigraph ??= ignored, use -trigraphs to enable [-Wtrigraphs]
 const void *s4 = R"abcdefghijklmnop??=(a)abcdefghijklmnop??=";
 ^
raw-string-14.c:23:58: warning: trigraph ??= ignored, use -trigraphs to enable [-Wtrigraphs]
raw-string-14.c:23:62 punctuator ;
raw-string-14.c:26:1 keyword const
raw-string-14.c:26:7 keyword void
raw-string-14.c:26:12 punctuator *
raw-string-14.c:26:13 identifier s5
raw-string-14.c:26:16 punctuator =
raw-string-14.c:26:18 identifier R
raw-string-14.c:26:19 string-literal "abc()abcdef"
raw-string-14.c:26:32 punctuator ;
raw-string-14.c:30:1 keyword const
raw-string-14.c:30:7 keyword void
raw-string-14.c:30:12 punctuator *
raw-string-14.c:30:13 identifier s6
raw-string-14.c:30:16 punctuator =
raw-string-14.c:30:18 identifier R
raw-string-14.c:30:19 string-literal "()"
raw-string-14.c:30:23 punctuator ;
raw-string-14.c:34:1 keyword const
raw-string-14.c:34:7 keyword void
raw-string-14.c:34:12 punctuator *
raw-string-14.c:34:13 identifier s7
raw-string-14.c:34:16 punctuator =
raw-string-14.c:34:18 identifier R
raw-string-14.c:34:19 string-literal "a()a"
raw-string-14.c:34:25 punctuator ;
raw-string-14.c:39:1 keyword int
raw-string-14.c:39:5 identifier main
raw-string-14.c:39:10 punctuator (
raw-string-14.c:39:11 punctuator )
raw-string-14.c:39:13 punctuator {
raw-string-14.c:39:14 punctuator }
